package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "nodemon":"nodemon index.js",
```

Comment:
Nodemon is only used in development so it make more sense to call the script `dev` or `start:dev`

Reply:
Got it, will edit it. thanks